calico/getting-started/kubernetes/requirements.mdx

```diff
@@ -10,7 +10,7 @@ description: Review requirements before installing Calico to ensure success.
 
 #### Supported versions
 
-We test {{prodname}} {{version}} against the following Kubernetes versions.
+We test {{prodname}} {{version}} against the following Kubernetes versions. Other versions may work, but we are not actively testing them.
```

Comment:
@ctauchen : I have included this new sentence in the docs for 3.25+. I wanted your input regarding this addition because I feel we have something really similar in lines 19-22 below. Can we combine/reword them entirely in some way ?